rfcs/048-concepts-rfcs/README.md

```diff
@@ -0,0 +1,45 @@
+# RFC 048: Concepts work plan
+
+**Status:** :construction: Draft :building_construction:
+
+**Last updated:** 29/04/2022
+
+## Context
+
+We are now starting work on introducing "concepts" to the Wellcome digital platform: identifiable entities like subjects, people, organisations, genres, etc. Works can be tagged with concepts, and concepts can be linked to other concepts by (for example) predicates, in this way forming a knowledge graph. There is a future possibility to infer a work's subjects, as well as manual tagging.
+
+This opens the door to a variety of possible user outcomes. A small number of these might be...
+
+- Dedicated, search-engine-indexable pages for (eg) subjects and people. These can possibly include curated content.
+- Searching by concept (rather than by keyword).
+- Navigating between concepts by their relationships to one another.
+- Having access to data on concepts from other sources (for example, Wikipedia) within the wc.org experience.
+
+The initial stage of this work will be to implement a minimal version of the first of these points: we will call these pages **concept pages**. These will likely not refer externally to "concepts"; instead it will be restricted to the aforementioned subjects and people.
+
+This RFC outlines the major technical decisions that will need to be made around data modelling, API design, and system architecture in order to serve this outcome - it does not aim to _make_ any of these decisions.
+
+## What needs to be done
+
+This approach presupposes the following architectural decisions:
+
+- There is a separate API endpoint within the catalogue for concepts 
+- There is a knowledge graph of concepts that exists independently of works
+- Concepts that works are tagged with are identifiable (the those identifiers exist in the graph) 
```

Comment:
not sure what you're getting at with the parenthetical

Reply:
Nor am I